ynput/OpenPype
# Maya: Simplify pointcache proxy integration

## Brief description
Missing commit for #4449 

This makes pointcache proxy and Arnold Scene Source proxy use the same integration workflow.

## Testing notes
1. Create pointcache in Maya and add meshes to the proxy objectset.
2. Publish.
3. Check the proxy alembic is integrated rather than ending up in the `resources` folder.
